parachain/pallets/outbound-queue/runtime-api/src/lib.rs

```diff
@@ -2,11 +2,14 @@
 // SPDX-FileCopyrightText: 2023 Snowfork <hello@snowfork.com>
 #![cfg_attr(not(feature = "std"), no_std)]
 
+use snowbridge_core::outbound::{Fees, Message};
 use snowbridge_outbound_queue_merkle_tree::MerkleProof;
 
 sp_api::decl_runtime_apis! {
 	pub trait OutboundQueueApi
```

Comment:
```suggestion
	pub trait OutboundQueueApi<Balance> where Balance: BalanceT {
```

Reply:
[Use Balance generic for the api](https://github.com/Snowfork/snowbridge/pull/975/commits/61d1df5e6b8a208e50998d19fde9a026d6df8afa)